src/cf.data.pre

```diff
@@ -2570,7 +2570,16 @@ DOC_START
 	Allows you to apply a Netfilter mark value to packets being transmitted
 	on the client-side, based on an ACL.
 
-	clientside_mark mark-value [!]aclname ...
+	clientside_mark mark[/mask] [!]aclname ...
+
+	Mark and mask are unsigned integers (hex, octal, or decimal).
+	The mask may be used to preserve marking previously set by other agents
+	(e.g., iptables).
+
+	The directive clears the original connmark's bits given by the mask and ORs
+	it with a new mark. The result is set as a new connmark. I.e. if a mask is
```

Comment:
We must polish this paragraph for clarity and to avoid the questions we had about similar iptables documentation. How about the following:

> A matching rule replaces the original connmark value with the configured mark. If a mask is also specified, then the masked bits of the original value are zeroed, and the configured mark is ORed with that adjusted value. For example, setting a 0xAB/0xF mark for a connection with a 0x5F connmark, results in a 0xFB connmark (rather than a 0xAB or 0x5B connmark).

It may be useful to also add the following paragraph:

> This directive semantics is similar to iptables --set-mark rather than --set-xmark functionality.

I also suggest adding a paragraph to document handling of multiple rules:

> The directive rules are evaluated in the configuration order. The first matching rule wins.